### ISSUE TITLE
Sort by Date / Timeline

### ISSUE THREAD

**Issue by jedwardwalters** [srophe/syriac-corpus-app#215]
@wsalesky None of the texts are encoded with date info yet. Any ideas about where that should be? I'm assuming in the tltleStmt somewhere. I can pick a few files for testing on dev server as soon as we nail down where the date should be encoded.

One thing we should think through: some texts may have specific dates, but many of them will just have a century (4th, 5th, etc.). I know we can encode a date with a range, like from=300 and to=400 for 4th century. Should we just encode all of them with that range, even when we have specific dates that fall within that range? In other words, what makes it easier on your end?

As for the Timeline, I don't really know how that works. So just let me know if I need to so something there.

**Comment by wsalesky** [comment]
How about the Publication Statement? As for the dates I need them to either be ISO format or at least easy to convert. I think it is best to use the @to @from @when attributes with ISO dates and then within the tag you can use any date you like, i.e. 4th or 5th century.

Here is an example of the timeline
http://wwwb.library.vanderbilt.edu/exist/apps/srophe/spear/index.html

More useful may be the date slider as seen here: 
https://architecturasinica.org/exist/apps/tcadrt/research-tool.html

Same needs as far as date formatting goes.

**Comment by jedwardwalters** [comment]
@davidamichelson @dlschwartz 
Any thoughts on how best to incorporate the timeline into the corpus?

**Comment by jedwardwalters** [comment]
And I think publication statement would be fine.

Will it work okay for sorting purposes if some of the texts are tagged to specific dates such as:
`<date when="337">337 CE</date>`
and others are only tagged to a century, such as:
`<date from="300" to="400">4th century CE</date>`

**Comment by jedwardwalters** [comment]
I do like the date slider thing. We'd also want there to be a box for date on the advanced search page.

**Comment by wsalesky** [comment]
Yes, although I think the correct ISO encoding is `<date when="0337">337 CE</date>`

**Comment by jedwardwalters** [comment]
Oh yes. Thank you for the reminder.

**Comment by jedwardwalters** [comment]
Okay, I have added some dates to a handful of texts for testing:
1.xml
11.xml
26.xml
27.xml
188.xml
228.xml
Note: these are only on the dev server.
